aclk/README.md

```diff
@@ -129,3 +129,28 @@ If you changed the runtime setting in your `var/lib/netdata/cloud.d/cloud.conf`
 Restart your Agent and [claim your node](/claim/README.md#how-to-claim-a-node).
 
 [![analytics](https://www.google-analytics.com/collect?v=1&aip=1&t=pageview&_s=1&ds=github&dr=https%3A%2F%2Fgithub.com%2Fnetdata%2Fnetdata&dl=https%3A%2F%2Fmy-netdata.io%2Fgithub%2Faclk%2FREADME&_u=MAC~&cid=5792dfd7-8dc4-476b-af31-da2fdb9f93d2&tid=UA-64295674-3)](<>)
+
+# ACLK and privacy
+
+Privacy is very important to us and we firmly believe that the data of our users belong to themselves. This is why **we actually don't store any metric data in the Netdata Cloud**. 
```

Comment:
Let's personalize this and stay away from abstractions like "the user." IMO that's for legal content, not documentation.

```suggestion
Privacy is very important to us. We firmly believe that your data belongs to you. This is why **we don't store any metric data in Netdata Cloud**. 
```